### DISCUSSION TITLE
Can't check or verify plots in CLI

### DISCUSSION THREAD

**Discussion by SackofFluor** [Chia-Network/chia-blockchain#4388] [Support]
Hey Guys,

I have a problem...
i have been plotting fine with only smaller problem here and there but now I wanted to use "chia plots check" and "chia plots verify" to check if my plots are actually alright and all is dandy, but whenever I try to use it with the CLI i get this error and the gui opens automatically.

**C:\Users\HOBO\AppData\Local\chia-blockchain\app-1.1.2>chia plots check

C:\Users\HOBO\AppData\Local\chia-blockchain\app-1.1.2>
C:\Users\HOBO\AppData\Local\chia-blockchain\app-1.1.2\resources\app.asar.unpacked\daemon
true
Error loading config - using defaults
startUrl file:///C:\Users\HOBO\AppData\Local\chia-blockchain\app-1.1.2\resources\app.asar\build\renderer\index.html
C:\Users\HOBO\AppData\Local\chia-blockchain\app-1.1.2\resources\app.asar.unpacked\daemon
true
C:\Users\HOBO\AppData\Local\chia-blockchain\app-1.1.2\resources\app.asar.unpacked\daemon
true
Running python executable:
child process success
(node:22136) electron: The default of contextIsolation is deprecated and will be changing from false to true in a future release of Electron.  See https://github.com/electron/electron/issues/23506 for more information
No cert
Have cert**

Anyone haven an idea?

**Comment by lvcivs** [comment]
First of all, you are running a pretty old release. It is recommended to upgrade to version 1.1.5, as a lot of bugs have been fixed.

The command looks right, though. What happens when you type `chia version`? It should show just the version number.

Strange that it would start the GUI.

**Reply by SackofFluor** [reply]
Thank you for the input. Since then I upgraded. But haven't tried again.
I will do so when this round of plotting is done!

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
